lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1511,7 +1511,9 @@ def cmd_set_help
     print_line "If both are omitted, print options that are currently set."
     print_line
     print_line "If run from a module context, this will set the value in the module's"
-    print_line "datastore.  Use -g to operate on the global datastore"
+    print_line "datastore.  Use -g to operate on the global datastore."
+    print_line
+    print_line "If setting a PAYLOAD, this command can take an index from `show payloads'."
```

Comment:
```suggestion
    print_line "If setting a PAYLOAD, this command can take an index from 'show payloads'."
```

Sneaky backtick.

Reply:
That is intentional.

Reply:
```suggestion  
    print_line "If setting a PAYLOAD, this command can take an index from `show payloads`."
```

Then I would like to see it matched on the other side of the command.

Reply:
```
wvu@kharak:~$ help | head -5
GNU bash, version 5.0.7(1)-release (x86_64-apple-darwin18.5.0)
These shell commands are defined internally.  Type `help' to see this list.
Type `help name' to find out more about the function `name'.
Use `info bash' to find out more about the shell in general.
Use `man -k' or `info' to find out more about commands not in this list.
wvu@kharak:~$
```

Reply:
https://github.com/v7unix/v7unix/blob/master/v7/usr/src/cmd/sh/main.c#L85

Reply:
```
wvu@kharak:~$ irb
irb(main):001:0> jeffrey
NameError: undefined local variable or method `jeffrey' for main:Object
	from (irb):1
	from /usr/bin/irb:11:in `<main>'
irb(main):002:0>
```

Reply:
Arguably, Python is on the right side of history:

```
wvu@kharak:~$ python3
Python 3.7.6 (default, Dec 30 2019, 19:38:28)
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> jeffrey
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'jeffrey' is not defined
>>>
```

And we're a bunch of curmudgeonly Unix users.

Reply:
¯\_(ツ)_/¯ still looks off to me, I will go with it though.

Reply:
Dammit, I was hoping you'd argue with me and convert me away from this hysterical raisin. :P

Reply:
It is too late in the evening for that, can discuss later and scrub the whole project for these.

Reply:
`s̔how payloads̓` may be a viable compromise.

Reply:
IoI